src/utils/PrimeSieve.ts

```diff
@@ -0,0 +1,51 @@
+/** Implementation of Sieve of Eratosthenes
```

Comment:
I think we can fold this into TotalPrimesInRange.ts, unless/until we come up with something else that needs to sieve for primes. (Which I find rather unlikely.)